packages/design-system-docs/src/pages/components/Dialog/_ThirdPartyLink.docs.scss

```diff
@@ -18,37 +18,29 @@ Style guide: patterns.external-link
 Style guide: patterns.external-link.react
 */
```

Comment:
Looks like if you remove the line `@react-props Dialog.tsx` above, it will remove the props table. It doesn't seem like we need a props table here, since it's just the dialog props table.

Reply:
We can link to the dialog